packages/components/src/index.js

```diff
@@ -33,6 +33,8 @@ export { default as FormFileUpload } from './form-file-upload';
 export { default as FormToggle } from './form-toggle';
 export { default as FormTokenField } from './form-token-field';
 export { default as __experimentalGradientPicker } from './gradient-picker';
+export { default as Guide } from './guide';
+export { default as GuidePage } from './guide/page';
```

Comment:
Storybook stories? :)

Reply:
how do I do that 🙈

Reply:
haha :) look at others `stories` folder (I think adding the folder and exporting a function from the index should be enough)

Reply:
Done! f8c494d80

<img width="1552" alt="Screen Shot 2019-11-28 at 14 42 13" src="https://user-images.githubusercontent.com/612155/69811002-6bb27400-11ed-11ea-999f-9089555924df.png">